app/assets/javascripts/admin/line_items/controllers/line_items_controller.js.coffee

```diff
@@ -11,54 +12,76 @@ angular.module("admin.lineItems").controller 'LineItemsCtrl', ($scope, $timeout,
   $scope.sharedResource = false
   $scope.columns = Columns.columns
   $scope.sorting = SortOptions
+  $scope.distributorFilter = ''
+  $scope.supplierFilter = ''
+  $scope.orderCycleFilter = ''
+  $scope.quickSearch = ''
+  $scope.page = 1
+  $scope.per_page = 50
 
   $scope.confirmRefresh = ->
     LineItems.allSaved() || confirm(t("unsaved_changes_warning"))
 
   $scope.resetSelectFilters = ->
-    $scope.distributorFilter = 0
-    $scope.supplierFilter = 0
-    $scope.orderCycleFilter = 0
-    $scope.quickSearch = ""
+    $scope.distributorFilter = ''
+    $scope.supplierFilter = ''
+    $scope.orderCycleFilter = ''
+    $scope.quickSearch = ''
+    $scope.page = 1
```

Comment:
these 2 occurences of these 5 lines above can be DRY'ed into a resetFilters method